packages/core/src/components/tooltip/tooltip.tsx

```diff
@@ -95,17 +95,11 @@ export interface ITooltipProps extends IProps, IIntentProps, IOverlayLifecyclePr
      */
     position?: Position | "auto";
 
-    /**
-     * The name of the HTML tag to use when rendering the popover target wrapper element (`Classes.POPOVER_WRAPPER`).
-     * @default "span"
-     */
-    rootElementTag?: string;
```

Comment:
why not let tooltips provide a `wrapperTagName`?

Reply:
i have a follow up PR to introduce `IPopoverSharedProps` that will resolve this.

Reply:
maybe i'll just open that now as there's a merge conflict here